horace_core/admin/cache.py

```diff
@@ -82,13 +81,11 @@ def read_chunk(filename,start,size,buf_size,progress,n_workers):
     if progress:
         prog_rep = progress_rep(size,n_workers)
         
-    fh = open(filename,'rb')
-    if fh<0:
-        raise ValueError("Can not open file: "+filename)
+    fh = open(filename,'rb',buffering=0)
     fh.seek(start,0)
     block = buf_size
     got   = 0
-    with open(filename, "rb") as f:
+    with fh as f:
```

Comment:
This is not standard Python style, does not handle errors properly.

Reply:
the errors with file opening've dealt above.